apps/test/integration/levelSolutions/turtle/4_11.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   levelId: '4_11',
   tests: [
     {
-      description: 'Top Solve: Repeat 9 times, right 80',
+      description: 'Top Solve: Repeat 2x, right 80',
```

Comment:
What does changing this do?

Reply:
nothing, i just noticed it was out of date -- we changed the code from a 9x repeat to a 2x repeat so i updated this to match